src/Pos/Launcher/Scenario.hs

```diff
@@ -111,6 +112,6 @@ initLrc = do
 initUSMemState :: WorkMode ssc m => m ()
 initUSMemState = do
     tip <- GS.getTip
-    tvar <- mvState <$> askUSMemVar
+    tvar <- mvState <$> askContext @UpdateContext ucMemState
```

Comment:
`askContext` is not very short, but okay for me, because it's clear. Other possibilities:
`askCtx` `askC` `ask'` `askEnv` `env` (like it, very similar to `ask`, short and clear) `context` `getEnv` `get` `fromEnv` `fromCtx`